src/CommunityToolkit.Maui/PopupService.cs

```diff
@@ -52,7 +52,18 @@ public PopupService()
 
 		ValidateBindingContext<TViewModel>(popup, out _);
 
+#if WINDOWS
```

Comment:
can you extract it to the single method and call only once? also could you please remove all null-forgiving operators. they are forbidden in this repo